nilearn/decoding/decoder.py

```diff
@@ -0,0 +1,849 @@
+"""High-level decoding object that exposes standard classification and
+regression strategies such as SVM, LogisticRegression and Ridge, with optional
+feature selection, and integrated hyper-parameter selection.
+"""
```

Comment:
It should be mentioned that it implements an aggregation strategy instead of standard selection.
I guess that this is reason why e.g. random forests are not supported ?

Reply:
@bthirion How do you wish to resolve this?

Reply:
sorry, in my comment above, selection was meant for 'model selection' nor 'feature selection'.